addons/source-python/packages/source-python/loggers.py

```diff
@@ -388,6 +390,10 @@ def areas(self):
         """
         return self._areas.get_int()
 
+    def _unload_instance(self):
+        """Remove the handler from the logger."""
+        self.logger.removeHandler(self._handler)
```

Comment:
Wouldn't it be better to delete the logger itself?
```python
self.logger.manager.loggerDict.pop(self.logger.name, None)
```
This would ensure handlers registered through `test_logger.logger.addHandler(...)` directly also get removed.

Reply:
That's certainly the appropriate way to do it!